exercises/robot_arm/solution/solution.js

```diff
@@ -6,20 +6,6 @@ board.on('ready', function () {
   var pot = new five.Sensor('A2')
 
   pot.on('change', function () {
-    var position = five.Fn.map(this.value,
-      0, 1023,
-      0,  179
-    )
-
-    servo.to(position)
-  })
-
-  // Alternatively, sensor provides a `scale` method as an alias to Fn.map
-  /*
-  pot.scale(0, 179).on('change', function () {
-
-    // `this.value` will reflect a scaling from 0-1023 to 0-179
-    servo.to(this.value)
+    servo.to(pot.scaleTo(0, 179))
```

Comment:
Don't we need to use `this.value` here?

Reply:
No, isn't necessary.

In this solution `this.value` doesn't have the scaled value.

If we want to use `this.value`, like in the `scale` comment, I think we have to chain the `scaleTo` with the on change: `pot.scaleTo(0, 179).on('change', ...)`

Reply:
ping @alanshaw